requirements/requirements.txt

```diff
@@ -4,5 +4,6 @@ ovos-utils~=0.0, >=0.0.35
 ovos-workshop~=0.0,>=0.0.15
 ovos-bus-client~=0.0
 padacioso~=0.1
+adapt-parser
```

Comment:
In ovos this was replaced with the adapt pipeline repo, it is a maintained fork of adapt + matching logic for ovos, this dependency should be dropped if you want to keep in sync